modules/core/src/main/java/org/apache/ignite/internal/binary/BinaryUtils.java

```diff
@@ -1044,8 +1044,12 @@ public static BinaryMetadata mergeMetadata(@Nullable BinaryMetadata oldMeta, Bin
                             "Type '" + oldMeta.typeName() + "' with typeId " + oldMeta.typeId()
                                 + " has a different/incorrect type for field '" + newField.getKey()
                                 + "'. Expected '" + oldFieldTypeName + "' but '" + newFieldTypeName
-                                + "' was provided. Field type's modification is unsupported, clean {root_path}/marshaller " +
-                                "and {root_path}/db/binary_meta directories if the type change is required."
+                                + "' was provided. Field type's modification is unsupported, if the type change is " +
```

Comment:
Taras, how about this version of the message that suggests a workaround? (I think it's enough to suggest the control.sh approach only skipping the option that explains how to clean the metadata manually):

_The type of an existing field can not be changed. Use a different field name or follow this procedure to reuse the current name: 
* Delete data records that use the old field type.
* Update the system metadata by running this command: 'control.sh/bat --meta remove --typeId oldMeta.typeId()'_

Reply:
Fixed